core/snapshot/config.go

```diff
@@ -39,16 +46,18 @@ func NewDefaultConfig() Config {
 		KeepRecent:  10,
 		RetryLimit:  5,
 		Storage:     LevelDB,
-		StartHeight: -1,
+		StartHeight: 0,
 	}
 }
 
 // Validate checks the values in the config file are sensible.
 func (c *Config) Validate() error {
-	// default to min 1 version, just so we don't have to account for nil slice.
-	// A single version kept in memory is pretty harmless.
-	if c.KeepRecent < 1 {
-		c.KeepRecent = 1
+	if c.KeepRecent < 1 || c.KeepRecent > 10 {
```

Comment:
I'm not sure of this restriction because it is useful forto able to keep more than 10 locally (even if 10 is the limit for listing during statesync). For example the `snapshot-soak-tests` rely on being able to keep hundreds of snapshots locally.